cluster/gce/config-test.sh

```diff
@@ -35,13 +35,23 @@ KUBE_APISERVER_REQUEST_TIMEOUT=300
 PREEMPTIBLE_NODE=${PREEMPTIBLE_NODE:-false}
 PREEMPTIBLE_MASTER=${PREEMPTIBLE_MASTER:-false}
 
+# Today the cluster startup scripts asssume the master and the nodes use the
+# same OS distro, thus same set of initialization instructions (e.g., metadata,
+# startup scripts, etc.). The current workaround is the hack in util.sh that
+# reloads <os_distro>/helper.sh in the gap between when the master is created
+# and when the nodes are created.
+# TODO(#26183): Provide a way to differentiate master OS distro and node OS
+# distro.
 OS_DISTRIBUTION=${KUBE_OS_DISTRIBUTION:-gci}
-# For GCI, leaving it blank will auto-select the latest GCI image on the `dev`
-# channel.
+# Note that NODE_IMAGE and NODE_IMAGE_PROJECT are kept to default to
+# MASTER_IMAGE and MASTER_IMAGE_PROJECT for backward compatibility. See
+# https://github.com/kubernetes/kubernetes/issues/26291 for details. GCI has to
+# set them differently, which is done in cluster/gce/util.sh. If you are
+# updating the verison of containervm, do it in that file.
```

Comment:
Seems unnecessary to me use five lines of comments for some implementation details